test/Bookstore.hs

```diff
@@ -405,7 +405,7 @@ mock (Model m) cmd = return $ case cmd of
 
 sm :: Bug -> StateMachine Model Command (ReaderT ConnectInfo IO) Response
 sm bug = StateMachine initModel transitions preconditions postconditions
-  Nothing generator shrinker (semantics bug) mock noCleanup
+  Nothing generator shrinker (semantics bug) mock noCleanup Nothing
```

Comment:
Doesn't the type of `semantics` need to change need to change to have `Tracer` in this and all other tests (except ticket dispenser and ioref)?